plugins/commands/powershell/command.rb

```diff
@@ -51,14 +59,14 @@ def execute
           end
 
           if !options[:command].nil?
-            out_code = machine.communicate.execute(options[:command].dup) do |type,data|
-              machine.ui.detail(data) if type == :stdout
-            end
-            if out_code == 0
-              machine.ui.success("Command: #{options[:command]} executed succesfully with output code #{out_code}.")
-            end
-            next
+            out_code = machine.communicate.execute(options[:command].dup, comops) do |type,data|
+            machine.ui.detail(data) if type == :stdout
+          end
+          if out_code == 0
```

Comment:
Based on these changes, won't `out_code` always be nil here? It seems like the only changes that should of been made here was adding `comops` to the `execute` function, but everything else should of stayed the same?